packages/genesys-spark-components/src/components/stable/gux-form-field/functional-components/gux-form-field-label-container/gux-form-field-label-container.tsx

```diff
@@ -0,0 +1,22 @@
+import { FunctionalComponent, h, VNode } from '@stencil/core';
+
+import { GuxFormFieldLabelPosition } from '../../gux-form-field.types';
+
+interface GuxFormFieldLabelContainerProps {
+  labelPosition: GuxFormFieldLabelPosition;
+}
+
+export const GuxFormFieldLabelContainer: FunctionalComponent<
+  GuxFormFieldLabelContainerProps
+> = ({ labelPosition }, children): VNode => {
+  return (
+    <div
+      class={{
+        'gux-form-field-label-container': true,
+        [`gux-${labelPosition}`]: true
+      }}
+    >
+      {children}
+    </div>
+  ) as VNode;
+};
```

Comment:
@daragh-king-genesys This could be done with one component instead of two. However, I thought we should keep it to make the implementation of form-field-label and form-field-legend-label the same. Open to other ideas on that